wtxmgr/tx.go

```diff
@@ -931,3 +981,114 @@ func (s *Store) Balance(ns walletdb.ReadBucket, minConf int32, syncHeight int32)
 
 	return bal, nil
 }
+
+// HasSpendingTx determines whether the store contains a spending transaction
+// for the given outpoint. This transaction can be either confirmed or
+// unconfirmed.
+func (s *Store) HasSpendingTx(ns walletdb.ReadBucket,
+	outpoint wire.OutPoint) (bool, *wire.MsgTx, error) {
```

Comment:
Can the bool return parameter be removed and a nil tx be used to indicated that there isn't a spending tx? I think the only case where you return true and a nil tx is when there is a serialization error. In that case return nil, err is good enough?

Reply:
Agreed- fixed!